src/main/java/net/minecraftforge/event/terraingen/WorldTypeEvent.java

```diff
@@ -114,9 +128,9 @@ public void setNewSize(int newSize)
         private final GenLayer[] originalBiomeGens;
         private GenLayer[] newBiomeGens;
 
-        public InitBiomeGens(WorldType worldType, long seed, GenLayer[] original)
+        public InitBiomeGens(@Nullable World world, WorldType worldType, long seed, GenLayer[] original)
```

Comment:
Public methods like this can't be changed directly, they need to be deprecated and replaced with new methods.